packages/gestalt/src/SegmentedControl.js

```diff
@@ -13,7 +13,7 @@ type OnChange = ({
   activeIndex: number,
 }) => void;
 
-type Props = {
+type Props = {|
```

Comment:
We don't need strict types. they were removed

try running yarn if it complains about it. See https://github.com/pinterest/gestalt/pull/3254